hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/translators/BaseTranslator.java

```diff
@@ -1018,6 +1019,13 @@ private void scanUnit(@NonNull final BlockTransactionalUnit unit) {
                     highestPutSerialNos
                             .computeIfAbsent(tokenId, ignore -> new LinkedList<>())
                             .add(nftId.serialNumber());
+                } else if (key.hasProtoBytesKey()) {
+                    final var keyBytes = key.protoBytesKeyOrThrow();
+                    final var value = mapUpdate.valueOrThrow();
+                    if (value.hasAccountIdValue()) {
+                        final var accountId = value.accountIdValueOrThrow();
+                        autoCreations.put(keyBytes, accountId);
```

Comment:
_Nit_ - I think this K/V state stores **all** aliases, not just those resulting from auto-creations.

Reply:
Changed to `aliases` . Thanks